src/components/shared/passphraseRenderer/index.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import fillWordsList from 'bitcore-mnemonic/lib/words/english';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { withTranslation } from 'react-i18next';
+import styles from './passphraseRenderer.css';
+import { PrimaryButton, TertiaryButton } from '../../toolbox/buttons/button';
+
+class PassphraseRenderer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      indexes: [2, 9],
+      fieldSelected: undefined,
+      displayOptions: undefined,
+      chosenWords: {},
+      options: {},
+      isCorrect: false,
+      hasErrors: false,
+      disabledButton: true,
```

Comment:
💅 I think it's not necessary to store this value, it can be easily computed from other state values

Reply:
✅

Reply:
I see you stopped using `disabledButton`, but forgot to remove this line
```suggestion
```

Reply:
✅